### PR TITLE
EOS-17214 Network partition - find a way to survive the partition running third party services

### PR DESCRIPTION
**Pull request by VijayThakkar132**
-----
[View rendered Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md](https://github.com/VijayThakkar132/cortx-experiments/blob/EOS-17214/Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md)

### CODE REVIEW

File: Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    dummy_third_party      (ocf::heartbeat:Dummy): Started srvnode-5Daemon Status:
+    corosync: active/enabled
+    pacemaker: active/enabled
+    pcsd: active/enabled
+    ```
+
+2. Modified quorum vote of the srvnode-5 from srvnode-1
+
+```bash
+corosync-quorumtool -v 5 -n 5
```
Comment (ajaysrivas) [inline Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md:68]: what is "-n" option?

Comment (ajaysrivas) [reply Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md:68]: Looks like a node id. Can't we use node names here?

Comment (VijayThakkar132) [reply Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md:68]: I don't think we can use node-name as it expects numerical value only. I had given try with node-name but it did not work.

Comment (ajaysrivas) [reply Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md:68]: How can we find node id from node name then? I mean, we need to find node id somehow.

Comment (VijayThakkar132) [reply Pacemaker/network-partition-survive-3rd-party-services/Increase-quorum-votes-to-survive-3rd-parties-services.md:68]: We can find out node-id using the list-nodes option **"corosync-quorumtool -l"** 

> [root@ssc-vm-c-1713 677029]# corosync-quorumtool -l

Membership information
----------------------
    Nodeid      Votes Name
         3          1 srvnode-3
         1          1 srvnode-1 (local)
         5          5 srvnode-5
         4          1 srvnode-4
         2          1 srvnode-2

**Pull request merged**